### ISSUE TITLE
Deprecated method 'ugettext_lazy'

### ISSUE THREAD

**Issue by adrian093** [charlesthk/django-simple-mail#16]
Method `ugettext_lazy` was deprecated on Django 3+ and [removed](https://docs.djangoproject.com/en/dev/internals/deprecation/#deprecation-removed-in-4-0) on Django 4. 
Please replace with `gettext_lazy`

Error:
```

File "/usr/local/lib/python3.10/site-packages/simple_mail/models.py", line 9, in <module>
     from django.utils.translation import ugettext_lazy as _
 ImportError: cannot import name 'ugettext_lazy' from 'django.utils.translation' (/usr/local/lib/python3.10/site-packages/django/utils/translation/__init__.py)
 
```

**Comment by fmilanese-1** [comment]
I have made a PR for this, hope it gets merged

**Comment by charlesthk** [comment]
Fixed in 2.3.5. Thanks @fmilanese-1
